wgengine/userspace.go

```diff
@@ -369,12 +370,16 @@ func (e *userspaceEngine) isLocalAddr(ip packet.IP) bool {
 // handleDNS is an outbound pre-filter resolving Tailscale domains.
 func (e *userspaceEngine) handleDNS(p *packet.ParsedPacket, t *tstun.TUN) filter.Response {
 	if p.DstIP == magicDNSIP && p.DstPort == magicDNSPort && p.IPProto == packet.UDP {
+		src := p.SrcIP.Bytes()
```

Comment:
You can do this without allocating.:

netaddr.IPv4(byte(p.SrcIP>>24),byte(p.SrcIP>>16),byte(p.SrcIP>>8), byte(p.SrcIP))

But I'd pull that out into a func so this is prettier here.

Reply:
Ah yes, my oversight.